citus.spec

```diff
@@ -43,8 +43,6 @@ make %{?_smp_mflags}
 # Install documentation with a better name:
 %{__mkdir} -p %{buildroot}%{pginstdir}/doc/extension
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
-%{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
```

Comment:
Why did we remove this NOTICE file here?

Reply:
Thanks for the catch. We have an open issue about updating templates. I need to update it manually until implementing this issue. I missed the manual check here